udemy_enroller/udemy_rest.py

```diff
@@ -220,10 +220,12 @@ def load_my_courses(self) -> List:
         all_courses = list()
         page_size = 100
 
-        my_courses = self.my_courses(1, page_size)
+        page = 1
+        my_courses = self.my_courses(page, page_size)
         all_courses.extend(my_courses["results"])
-        total_pages = my_courses["count"] // page_size
-        for page in range(2, total_pages + 2):
+        
+        while ("next" in my_courses and my_courses["next"] != None):
```

Comment:
There's no need for the parentheses and the check against None should use `is not`"
`while "next" in my_courses and my_courses["next"] is not None`